tensorflow/compiler/tf2tensorrt/kernels/trt_engine_resource_ops.cc

```diff
@@ -228,6 +236,21 @@ class SerializeTRTResource : public OpKernel {
         engine_instance.set_serialized_engine(engine_data->data(),
                                               engine_data->size());
 
+        if (export_trt_engines_env) {
+          const std::string engine_filename = std::string(export_trt_engines_env) + "/" + resource_name;
+          std::unique_ptr<WritableFile> engine_file;
+          OP_REQUIRES_OK(ctx, ctx->env()->NewWritableFile(engine_filename, &engine_file));
+          engine_file->Append(StringPiece(static_cast<char*>(engine_data->data()), engine_data->size()));
+
+          const std::string dims_filename = std::string(export_trt_engines_env) + "/dims-" + resource_name;
+          std::unique_ptr<WritableFile> dims_file;
+          OP_REQUIRES_OK(ctx, ctx->env()->NewWritableFile(dims_filename, &dims_file));
+
+          for (const TensorShape& shape : engine_input_shapes) {
+            dims_file->Append(StringPiece(shape.DebugString()));
```

Comment:
I got this error for the Append function call:
 error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]
          engine_file->Append(StringPiece(

Reply:
Ack -- will look into it @bixia1.